src/Tree.js

```diff
@@ -29,6 +29,7 @@ export default class Tree extends Viz {
 
     this._shape = constant("Circle");
     this._shapeConfig = assign(this._shapeConfig, {
+      ariaLabel: (d, i) =>  this._treeData ? `${this._treeData[i].depth}. ${this._drawLabel(d, i)}, ${d.value}.` : "",
```

Comment:
We cannot use `d.value` directly here, as the user may have different data. Because Tree does not necessarily have numerical values associates to each node, just remove it from the ariaLabel entirely.